### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by miao0miao**
This guide will replace the contribution-git.md 
It includes more ways to participate besides coding.

There are a few more loose ends here:

- Pull Request Lifecycle - TBD, not sure if we need that but perhaps we should create a n overview of our workflow here 
- Development Environment Setup - TBD, this could be a place to describe how testing can be done. 
- Report bugs and request features - we can consider adding another label for more experienced community members like "help needed" and not just "first good issue"

### CODE REVIEW

File: CONTRIBUTING.md
Comment (dongsupark) [inline CONTRIBUTING.md]: Not a new issue. In the past it was discussed a few times.
Anyway In my opinion, it is not a good idea to list all possible timezone entries for a meeting time like  that. When the next winter time starts in Oct., we have to fix all the entries in this doc again, and repeat the same every year in Spring and Autumn. Same story for the Flatcar website. Huge maintenance burden.
That's why I always suggest writing only UTC(GMT).

File: CONTRIBUTING.md
Comment (dongsupark) [inline CONTRIBUTING.md]: the same

File: CONTRIBUTING.md
Comment (t-lo) [inline CONTRIBUTING.md:46]:
```suggestion
* Evangelise the project in talks, presentations, and workshops
* Coordinate work between Flatcar maintainers and upstream projects, for instance coordinating work items and planning project roadmaps in special sub-projects like Flatcar CAPI, or our sysext initiative
* Work with contributors and maintainers to organise project-wide events like bug fixing or doc writing days, devrooms at conferences, community days / meetups, or even conferences

```

File: CONTRIBUTING.md
Comment (t-lo) [inline CONTRIBUTING.md]:
```suggestion
[meetings](### Monthly Office hours and Developer Syncs) or [contact us](maintainers@flatcar-linux.org) and let's discuss how we can work
```

File: CONTRIBUTING.md
Comment (t-lo) [inline CONTRIBUTING.md]:
```suggestion
**Flatcar Office Hours are on the second Tuesday of every month at 5:30pm CE[S]T**

The meeting time observes the Central European Time zone and is subject to summer time changes.
It occurs at 5:30pm CE[S]T.
* During summer time (CEST), the meeting occurs at 9pm IST / 5:30pm CEST / 3:30pm GMT / 11:30am EDT / 8:30am PST.
* Outside of summer time (CET), the meeting occurs at 10pm IST / 5:30pm CEST / 4:30pm GMT / 11:30am EST / 8:30am PST.
Special cases occur when CEST is active and EDT/PDT are not - which usually occurs in a narrow, 2-week time window each spring and autumn. Since the meeting always observes CE[S]T, special care needs to be taken by attendees from the Americas.


Meeting agendas are published in advance - check our [discussions section](../../discussions/categories/office-hours-agenda) for examples.
* Video call link: [https://meet.flatcar.org/OfficeHours](https://meet.flatcar.org/OfficeHours)
* A Youtube live stream (which also serves as the meeting's recording) will be published on the respective agenda when a meeting starts.

**Flatcar Developer Syncs commence every 4th Tuesday of a month at 5:30pm CE[S]T **

The meeting time observes the Central European Time zone and is subject to summer time changes.
It occurs at 5:30pm CE[S]T.
Please see above for implications.

File: CONTRIBUTING.md
Comment (t-lo) [inline CONTRIBUTING.md]:
```suggestion
For an introduction to the Flatcar SDK and a walk-through of common developer cases like customising the OS image (e.g. adding or upgrading packages), have a look at our [developer guides](https://www.flatcar.org/docs/latest/reference/developer-guides/); particularly the [howto on building custom images from source](https://www.flatcar.org/docs/latest/reference/developer-guides/sdk-modifying-flatcar/).
The guides aim to provide a solid base for working with the SDK to help you filing successful PRs to the Flatcar project.
```

File: CONTRIBUTING.md
Comment (t-lo) [inline CONTRIBUTING.md]:
```suggestion
```
Remove this here and move it to "development environment" below.

File: CONTRIBUTING.md
Comment (t-lo) [inline CONTRIBUTING.md]:
```suggestion

Pull requests may be issued from repository branches or from a fork of the repo.
Branch PRs are limited to Maintainers since these require write access to the respective repository.
While the origin of a PR may have impact on the approvals required for the CI to build and test, the project does not discriminate PR origins regarding getting a change reviewed or merged.

PRs need to undergo a successful build+test as well as a review, and require at least one LGTM from a maintainer who is not the author of the PR for a merge.
Note that the approver(s) of a PR may be co-authors though - this allows reviewers to suggest changes.

A PR undergoes the following stages:
1. PR filed. <br/>
    This includes filing "draft" PRs which are not considered ready for review and will only undergo build+test  when explicitly requested.
2. PR is ready for review. <br/>
    In this stage, maintainers can start reviewing the PR as well as approve build+test runs.
    PRs can be filed in this stage if the author considers the PR ready at the time of creation.
    To generate traction on stale PRs, authors or project coordinators may actively reach out to the maintainers team, e.g via the Matrix channel or in office hours and developer syncs.
3. PR is being reviewed. <br/>
  Maintainers add comments and may request changes.
  Reviewers may also add change suggestions, streamlining the review process for the author.
  During review, the PR is also vetted against Flatcar's mission statement and our core principles.
  Multiple build + test runs may be approved by maintainers to further validate the PR.
4. PR is being merged, or closed w/o merge. <br/>
  Depending on the outcome of the review, the PR is being merged or closed without a merge.
```

File: CONTRIBUTING.md
Comment (t-lo) [inline CONTRIBUTING.md:55]:
```suggestion

You can reach contributors and maintainers via

* Our matrix chat (via element.io): https://app.element.io/#/room/#flatcar:matrix.org
* Our Slack channel in the Kubernetes Slack org: https://kubernetes.slack.com/archives/C03GQ8B5XNJ
* By engaging in or starting a new Github Discussion: https://github.com/flatcar/Flatcar/discussions

Though Matrix and Github discussions are encouraged and the preferred way to interact with the Flatcar community, we also maintain mailing lists for Flatcar users and developers:
* Flatcar Users: https://groups.google.com/g/flatcar-linux-user
* Flatcar Devs: https://groups.google.com/g/flatcar-linux-dev


```

**Review by t-lo — CHANGES_REQUESTED** [review]
Updated the document to reflect the project's current processes.

**Comment by t-lo** [comment]
Marking as ready for review, which the PR should be after the suggestions are merged.

**Review by t-lo — APPROVED** [review]
LGTM.

**Pull request merged**
